docs/source/contribute.rst

```diff
@@ -100,8 +100,84 @@ the standard. For example, following variable names are accepted:
 * ``w,x,y``: for representing weight, input, output tensors
 * ``_``: for unused variables
 
+Contributing New Models as Examples
+-----------------------------------
+
+To contribute a new model within a specific supported tensor framework (e.g. PyTorch, or MXNet), simply
+
+1. Make a directory with the name of your model (say ``awesome-gnn``) within the directory
+   ``examples/${DGLBACKEND}`` where ``${DGLBACKEND}`` refers to the framework name.
+   
+2. Populate it with your work, and make a pull request once you are done.
+
+   * Instructions for running your program and the results are preferred in your README.
```

Comment:
Populate it with your work, and make a pull request once you are done.
. Please provide a README with at least two sections. First, instructions for running your program. Second, the performance results such as speed or accuracy compared with some alternative implementations.